.travis.yml

```diff
@@ -0,0 +1,57 @@
+language: android
+jdk: oraclejdk8
+sudo: true
+env:
+  matrix:
+  - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
```

Comment:
This is a travis config for preinstalled components: https://docs.travis-ci.com/user/languages/android/#Pre-installed-components

I tried 'android-25' but that wasn't available.  I'm guessing this is to indicate the min required components needed for the build.   This may not even be required.  I'll test without it to see if it makes any difference.